source/api_documentation/network_integration/networks/_create_users.rst

```diff
@@ -1,38 +0,0 @@
-.. container:: endpoint-long-description
```

Comment:
@mikeweaver 
Did we have any kind of convention going for the collapsable GET / POST / PUT sections?
It looks like ```_[http_method]_[description].rst``` is the convention.

If that is the case, it looks like `network_integration/networks` had some unused files. There are 3 collapsable sections but 5 files. `_create_users.rst` is identical to `_post_network_users.rst` except for a newline.
I'd like to delete those to avoid confusing people.